package.nls.json

```diff
@@ -11,7 +11,6 @@
     "AWS.cloud9.commandPalette": "Go to Anything Panel",
     "AWS.cloud9.codelens": "Inline Action",
     "AWS.cloud9.codelenses": "Inline Actions",
-    "AWS.failedToWrite": "{0}: Failed to write \"{1}\". Use the \"{2}\" command to see error details.",
```

Comment:
Was it a conscious decision to remove these (since you're editing all of their contents)? I acknowledge that we can now write from source to the nls file, but shouldn't that be run prior to commit so our repo has an up-to-date reflection of the nls file?

Reply:
The `package.nls.json` should _only_ contain strings found in `package.json`
Any other string will be removed eventually, either by a script we write or manually.

Reply:
This doesn't make sense if we ever get localizations. Do we expect localization teams to dig through the code to find strings to replace?

Reply:
> This doesn't make sense if we ever get localizations. Do we expect localization teams to dig through the code to find strings to replace?

No there will be localization files. Just this file in particular only applies to static delcarations in package.json